src/snowflake/snowpark/session.py

```diff
@@ -1623,8 +1623,14 @@ def query_history(self) -> QueryHistory:
         return query_listener
 
     def _table_exists(self, table_name: str):
```

Comment:
```suggestion
    def _table_exists(self, table_name: str) -> bool:
```

Reply:
Damn I forgot....Thanks for catching it (again).